storybook/src/components/Header/Header.stories.tsx

```diff
@@ -3,14 +3,58 @@
  * Copyright Gemeente Amsterdam
  */
 
-import { PageMenu } from '@amsterdam/design-system-react'
+import { PageMenu, Screen } from '@amsterdam/design-system-react'
 import { Header } from '@amsterdam/design-system-react/src'
-import { SearchIcon } from '@amsterdam/design-system-react-icons'
+import { MenuIcon, SearchIcon } from '@amsterdam/design-system-react-icons'
 import { Meta, StoryObj } from '@storybook/react'
 
+const MenuButton = (
+  <PageMenu.Item>
+    <PageMenu.Button icon={MenuIcon} onClick={() => {}}>
+      Menu
+    </PageMenu.Button>
+  </PageMenu.Item>
+)
+
+const SearchButton = (
+  <PageMenu.Item>
+    <PageMenu.Button icon={SearchIcon} onClick={() => {}}>
+      Zoeken
+    </PageMenu.Button>
+  </PageMenu.Item>
+)
+
+const SecondaryLinks = (
+  <>
+    <PageMenu.Item>
+      <PageMenu.Link href="#" lang="en" rank="secondary">
+        English
+      </PageMenu.Link>
+    </PageMenu.Item>
+    <PageMenu.Item>
+      <PageMenu.Link href="#" rank="secondary">
+        Mijn Amsterdam
+      </PageMenu.Link>
+    </PageMenu.Item>
+  </>
+)
+
 const meta = {
   title: 'Components/Containers/Header',
   component: Header,
+  argTypes: {
+    as: {
+      control: { type: 'radio' },
+      options: ['header', 'div'],
+    },
```

Comment:
In case you every want to add a story that opens the menu? 

```suggestion
export const WithPageMenu: Story = {
```